package.json

```diff
@@ -20,23 +20,32 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
-    "eslint": "^4.19.1",
+    "eslint": "^5.1.0",
     "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-mocha": "^5.0.0",
-    "eslint-plugin-prettier": "^2.6.0",
+    "eslint-plugin-mocha": "^5.1.0",
+    "eslint-plugin-prettier": "^2.6.2",
     "mailparser": "^2.2.0",
     "mocha": "^5.2.0",
-    "prettier": "^1.13.5",
+    "prettier": "^1.13.7",
+    "rollup": "^0.62.0",
+    "rollup-plugin-commonjs": "^9.1.3",
+    "rollup-plugin-node-resolve": "^3.3.0",
     "smtp-server": "^3.4.6"
   },
   "engine": [
     "node >= 6"
   ],
   "main": "email.js",
   "scripts": {
-    "test": "mocha -R spec"
+    "test": "mocha",
+    "rollup": "rollup -c rollup.config.js && mocha --file email.bundle.test.js"
```

Comment:
could these be separated to a "rollup" and "rollup-test"? or is it preferred to have it happen in one step?

Reply:
done :)